docs/admin/kubeadm.md

```diff
@@ -13,188 +13,245 @@ Running `kubeadm init` bootstraps a Kubernetes cluster. This consists of the
 following steps:
 
 1. kubeadm runs a series of pre-flight checks to validate the system state
-before making changes. Some checks only trigger warnings, others are
-considered errors and will exit kubeadm until the problem is corrected or
-the user specifies `--skip-preflight-checks`.
+   before making changes. Some checks only trigger warnings, others are
+   considered errors and will exit kubeadm until the problem is corrected or the
+   user specifies `--skip-preflight-checks`.
 
 1. kubeadm generates a token that additional nodes can use to register
-themselves with the master in future.  Optionally, the user can provide a token.
+   themselves with the master in future.  Optionally, the user can provide a
+   token.
 
-1. kubeadm generates a self-signed CA using openssl to provision identities
-for each node in the cluster, and for the API server to secure communication
-with clients.
+1. kubeadm generates a self-signed CA using openssl to provision identities for
+   each node in the cluster, and for the API server to secure communication with
+   clients.
 
 1. Outputting a kubeconfig file for the kubelet to use to connect to the API
-server, as well as an additional kubeconfig file for administration.
+   server, as well as an additional kubeconfig file for administration.
 
 1. kubeadm generates Kubernetes resource manifests for the API server,
-controller manager and scheduler, and placing them in
-`/etc/kubernetes/manifests`. The kubelet watches this directory for static
-resources to create on startup. These are the core components of Kubernetes, and
-once they are up and running we can use `kubectl` to set up or manage any
-additional components.
+   controller manager and scheduler, and placing them in
+   `/etc/kubernetes/manifests`. The kubelet watches this directory for static
+   resources to create on startup. These are the core components of Kubernetes,
+   and once they are up and running we can use `kubectl` to set up or manage any
+   additional components.
 
```

Comment:
in between here kubeadm taints the master unschedulable, sets up RBAC, creates the cluster-info configmap for the token discovery and deploys both kube-proxy and kube-dns